org.midori_browser.Midori.json

```
@@ -0,0 +1,50 @@
+{
+  "app-id": "org.midori_browser.Midori",
+  "branch": "stable",
+  "runtime": "org.gnome.Platform",
+  "runtime-version" : "3.32",
+  "sdk": "org.gnome.Sdk",
+  "command": "midori",
+  "finish-args" : [
+    "--device=dri",
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--filesystem=xdg-download:rw",
+    "--talk-name=org.freedesktop.Notifications",
+    "--system-talk-name=org.freedesktop.GeoClue2",
+    "--talk-name=org.freedesktop.secrets"
+  ],
+  "rename-icon": "midori",
+  "rename-appdata-file": "midori.appdata.xml",
+  "modules": [
+    {
+      "name": "libpeas",
+      "cleanup": [
+        "/bin/*",
+        "/lib/peas-demo"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://gitlab.gnome.org/GNOME/libpeas.git",
+          "tag": "libpeas-1.22.0"
+        }
+      ]
+    },
+    {
+      "name": "midori",
+      "buildsystem": "cmake",
```

```suggestion
      "buildsystem": "cmake-ninja",
```

(may or may not work, but would be a bit faster)

---

Ninja should work fine actually.

---

I'll make the change